### PR TITLE
Adding a tutorial video link I made to the readme to use DAAM script …

### PR DESCRIPTION
**Pull request by FurkanGozukara**
…very easily

This tutorial video is based on the most famous Stable Diffusion open source GUI Automatic1111

It uses DAAM extension

I hope you accept this merge. Thank you very much

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,10 @@ Our datasets are here: https://git.uwaterloo.ca/r33tang/daam-data
 ## See Also
 - [1littlecoder's video](https://www.youtube.com/watch?v=J2WtkA1Xfew) for a code demonstration and Colab notebook of an older version of DAAM.
 
+- [Easiest way to use DAAM script tutorial](https://www.youtube.com/watch?v=XiKyEKJrTLQ)
+
+[![image.png](https://s3.amazonaws.com/moonup/production/uploads/1675628788246-6345bd89fe134dfd7a0dba40.png)](https://www.youtube.com/watch?v=XiKyEKJrTLQ)
```
Comment (daemon) [inline README.md]: Please remove this image.

**Review by daemon — CHANGES_REQUESTED** [review]
Looks good except for the image, thanks for the PR.

File: README.md (hunk 1 above)
Comment (FurkanGozukara) [reply README.md]: @daemon thank you so much. removed the image as requested. sorry for being late

**Review by daemon — APPROVED** [review]
Thanks!

**Pull request merged**